dist/docker/debian/build_docker.sh

```diff
@@ -74,10 +74,10 @@ bcp dist/docker/scylla_bashrc /scylla_bashrc
 
 run apt-get -y clean expire-cache
 run apt-get -y update
-run apt-get -y install dialog apt-utils
+run apt-get -y --no-install-suggests install dialog apt-utils
 run bash -ec "echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections"
 run bash -ec "rm -rf /etc/rsyslog.conf"
-run apt-get -y install hostname supervisor openssh-server openssh-client openjdk-11-jre-headless python2 python3 python3-yaml curl rsyslog sudo
+run apt-get -y --no-install-suggests install hostname supervisor openssh-server openssh-client openjdk-11-jre-headless python2 python3 python3-yaml curl rsyslog sudo
```

Comment:
[offtopic] it's surprising that we are still installing python2. @yaronkaikov hi Yaron, i see you added python3 in 9f9ee8a812cf39cdf3a85b7c747f254f31b2fdaf but somehow kept python2.7 around. can we drop python2 now?

Reply:
Quick search - https://github.com/scylladb/seastar/blob/665f3c20c420eaa0bfa4db32c4d9ba3fd81a22ed/scripts/dpdk_nic_bind.py#L1

Reply:
hmm, this script was copied from dpdk. and the dpdk submodule has migrated to python3 years ago. we can even drop it and call the one in dpdk submodule from our helper script. i am just too lazy to setup a dpdk env to do this -- i tried 1 year ago, but failed somehow. so this can be dropped.

Reply:
Drop this script you mean?

Reply:
to be more specific: we can either 

- review and merge https://github.com/scylladb/seastar/pull/1529,
- or
  1. drop the script and
  2. update https://github.com/scylladb/seastar/blob/master/scripts/run_with_dpdk.sh to use the script in the dpdk submodule.